generate-workflow.py

```diff
@@ -199,7 +200,7 @@ class BenchmarkInfo(NamedTuple):
 # generate-workflow.py instead and re-run it. Some things in this file are
 # explained by comments in generate-workflow.py.
 
-name: 3C benchmark tests
+name: {workflow.name}
 
 on:
   # Run every day at 3:00 AM EDT (i.e., 7 am UTC)
```

Comment:
The schedule times of the two workflows should probably be spaced well apart because of #6.